pkg/controller/migmigration/task.go

```diff
@@ -457,8 +457,10 @@ func (t *Task) Run() error {
 		if err != nil {
 			return liberr.Wrap(err)
 		}
-		if err = t.next(); err != nil {
-			return liberr.Wrap(err)
+		if t.Requeue != FastReQ {
```

Comment:
@JaydipGabani I don't think we should be using `t.Requeue` as a signal for whether `t.annotateStageResources()` is done.

Reply:
I suggest adapting `t.annotateStageResources()` to directly return a value indicating  if it has completed all annotations.

```
finished, err := t.annotateStageResources()
// if !finished, allow default FastReQ to be used
// if finished, proceed to next phase with `t.next()`
```